packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2898,10 +2898,10 @@ describe('<Autocomplete />', () => {
     );
     expect(handleHighlightChange.callCount).to.equal(
       // FIXME: highlighted index implementation should be implemented using React not the DOM.
-      reactMajor >= 18 ? 2 : 1,
+      reactMajor > 18 ? 2 : 1,
```

Comment:
This is due to this change: https://github.com/mui/material-ui/pull/44873.

I'm not sure why React 18 stopped sending two events and React 19 didn't, but I suspect it's the difference in strict mode handling.

Anyway, the important thing is that the event callback is called and that the arguments are correct, which still holds true 👍🏼